### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'line') at Object.fixFaultyLocations (node_modules/recast/lib/util.js:153:23)

### ISSUE THREAD

**Issue by 0xdevalias** [pionxzh/wakaru#51]
Attempting to unpack/unminify some webpacked code ([Ref](https://github.com/0xdevalias/chatgpt-source-watch/blob/main/unpacked/_next/static/chunks/pages/_app.js)):

```js
⇒ cd ./unpacked/_next/static/chunks

⇒ npx @wakaru/unpacker pages/_app.js -o ./pages/_app-unpacked/
Generated 213 modules from pages/_app.js to pages/_app-unpacked (24,049ms)

⇒ npx @wakaru/unminify ./pages/_app-unpacked/* -o ./pages/_app-unminified
# ..snip..
```

I got the following error while attempting to unminify:

```
# ..snip..
• Transforming pages/_app-unminified/pages/_app-unpacked/module-76984.js (203.7ms)
TypeError: Cannot read properties of undefined (reading 'line')
    at Object.fixFaultyLocations (/Users/devalias/dev/0xdevalias/REDACTED/node_modules/recast/lib/util.js:153:23)
    at TreeCopier.TCp.copy (/Users/devalias/dev/0xdevalias/REDACTED/node_modules/recast/lib/parser.js:138:10)
    at TreeCopier.TCp.copy (/Users/devalias/dev/0xdevalias/REDACTED/node_modules/recast/lib/parser.js:190:30)
    at TreeCopier.TCp.copy (/Users/devalias/dev/0xdevalias/REDACTED/node_modules/recast/lib/parser.js:190:30)
    at TreeCopier.TCp.copy (/Users/devalias/dev/0xdevalias/REDACTED/node_modules/recast/lib/parser.js:190:30)
    at TreeCopier.TCp.copy (/Users/devalias/dev/0xdevalias/REDACTED/node_modules/recast/lib/parser.js:190:30)
    at TreeCopier.TCp.copy (/Users/devalias/dev/0xdevalias/REDACTED/node_modules/recast/lib/parser.js:190:30)
    at TreeCopier.TCp.copy (/Users/devalias/dev/0xdevalias/REDACTED/node_modules/recast/lib/parser.js:190:30)
    at TreeCopier.<anonymous> (/Users/devalias/dev/0xdevalias/REDACTED/node_modules/recast/lib/parser.js:131:30)
    at Array.forEach (<anonymous>)
• Transforming pages/_app-unminified/pages/_app-unpacked/module-7718.js (2,651.5ms)
# ..snip..
```

**Comment by pionxzh** [comment]
This is not stable to reproduce, it only happened on the CLI... Still investigating

**Comment by pionxzh** [comment]
This issue is either from `lebab` or `recast` or `espree`.
The main issue is with the code `new.target`.

See https://github.com/lebab/lebab/issues/352

**Comment by pionxzh** [comment]
This seems fixed by the new version of `lebab`, or by some mystery factor. I can run the sample without error now.

**Comment by pionxzh** [comment]
Hold on. I can still reproduce the error in a smaller sample, bet ye, not sure why the CLI is working fine now. Will track this in lebab side.

**Comment by pionxzh** [comment]
I have submitted a [PR](https://github.com/lebab/lebab/pull/354) to lebab to allow passing a custom parser, which can fix this issue by using the babel parser.

**Comment by pionxzh** [comment]
Patched recast 😓

**Comment by pionxzh** [comment]
released #103
